lib/cli.js

```diff
@@ -130,6 +130,111 @@ program
     })
   })
 
+// provider ios
+program
```

Comment:
Now that we don't have all plugins, there are options that we don't need at this time.